examples/vision/md/image_captioning.md

```diff
@@ -266,21 +266,11 @@ def process_input(img_path, captions):
 
 
 def make_dataset(images, captions):
-    if split == "train":
-        img_dataset = tf.data.Dataset.from_tensor_slices(images).map(
-            read_train_image, num_parallel_calls=AUTOTUNE
-        )
-    else:
-        img_dataset = tf.data.Dataset.from_tensor_slices(images).map(
-            read_valid_image, num_parallel_calls=AUTOTUNE
-        )
-
-    cap_dataset = tf.data.Dataset.from_tensor_slices(captions).map(
-        vectorization, num_parallel_calls=AUTOTUNE
-    )
+    dataset = tf.data.Dataset.from_tensor_slices((images, captions))
+    dataset = dataset.shuffle(len(images))
```

Comment:
Full-shuffle will be expensive. Prefer something like `shuffle(BATCH_SIZE * 8)`

Reply:
`image captioning.py` and `image captioning.ipynb` consists of the following code:

```python
def make_dataset(images, captions):
    dataset = tf.data.Dataset.from_tensor_slices((images, captions))
    dataset = dataset.shuffle(len(images))
    dataset = dataset.map(process_input, num_parallel_calls=AUTOTUNE)
    dataset = dataset.batch(BATCH_SIZE).prefetch(AUTOTUNE)

    return dataset
```

This is the following code of `image_captioning.md` before PR:

```python
def make_dataset(images, captions):
    if split == "train":
        img_dataset = tf.data.Dataset.from_tensor_slices(images).map(
            read_train_image, num_parallel_calls=AUTOTUNE
        )
    else:
        img_dataset = tf.data.Dataset.from_tensor_slices(images).map(
            read_valid_image, num_parallel_calls=AUTOTUNE
        )

    cap_dataset = tf.data.Dataset.from_tensor_slices(captions).map(
        vectorization, num_parallel_calls=AUTOTUNE
    )

    dataset = tf.data.Dataset.zip((img_dataset, cap_dataset))
    dataset = dataset.batch(BATCH_SIZE).shuffle(256).prefetch(AUTOTUNE)
    return dataset

```
As you notice, they are different, and `image_captioning.md` gives a NameError.

Am I supposed to update the `shuffle(BATCH_SIZE * 8)` in all the 3 files or just `.md`?

Thank you,

Reply:
> Am I supposed to update the shuffle(BATCH_SIZE * 8) in all the 3 files or just .md?

All files should be kept consistent.

Reply:
Fixed @fchollet